crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -243,7 +243,7 @@ pub(crate) trait StateWriteExt: StateWrite {
     #[instrument(skip(self))]
     async fn put_deposit_event(&mut self, deposit: Deposit) -> Result<()> {
         let nonce = self.get_deposit_nonce(deposit.rollup_id()).await?;
-        self.put_deposit_nonce(deposit.rollup_id(), nonce + 1);
+        self.put_deposit_nonce(deposit.rollup_id(), nonce.wrapping_add(1));
```

Comment:
Would this too maybe be better as a `checked_add().unwrap()`?

Reply:
Maybe not a panic, but I think this should definitely be a `checked_add` because if nonces exceed that we have a problem.
```suggestion
        self.put_deposit_nonce(
            deposit.rollup_id(),
            nonce.checked_add(1).context("nonce overflowed")?
        );
```

Reply:
Done in [44c99af](https://github.com/astriaorg/astria/pull/1081/commits/44c99affc52d66ee3134fa1bbc23f66d21bb632a).